specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc

```diff
@@ -1346,43 +1343,43 @@ report an error. Application client modules are initialized on their own
 schedule, typically when an end user invokes them; as such, they are
 excluded from any initialization ordering requirements.
 
-=== [[a3203]]Java EE Application XML Schema
+=== [[a3203]]Jakarta EE Application XML Schema
 
-The XML grammar for a Java EE application
-deployment descriptor is defined by the Java EE application schema. The
+The XML grammar for a Jakarta EE application
+deployment descriptor is defined by the Jakarta EE application schema. The
 root element of the deployment descriptor for a Jaav EE application is
-_application_ . The granularity of composition for Java EE application
-assembly is the Java EE module. A Java EE application deployment
+_application_ . The granularity of composition for Jakarta EE application
+assembly is the Jakarta EE module. A Jakarta EE application deployment
 descriptor contains a name and description for the application and the
-URI of a UI icon for the application, as well a list of the Java EE
+URI of a UI icon for the application, as well a list of the Jakarta EE
 modules that comprise the application. The content of the XML elements
 is in general case sensitive. This means, for example, that
 _<role-name>Manager</role-name>_ is a different role than
 _<role-name>manager</role-name>_ .
 
-All valid Java EE application deployment
+All valid Jakarta EE application deployment
 descriptors must conform to the XML Schema definition, or the DTD or
 schema definition from a previous version of this specification. (See
 link:#a3447[See Previous Version
 Deployment Descriptors].”) The deployment descriptor must be named
 _META-INF/application.xml_ in the _.ear_ file. Note that this name is
 case-sensitive. The XML Schema located at
-_http://xmlns.jcp.org/xml/ns/javaee/application_8.xsd_ defines the XML
-grammar for a Java EE application deployment descriptor.
+_http://xmlns.jcp.org/xml/ns/jakartaee/application_8.xsd_ defines the XML
+grammar for a Jakarta EE application deployment descriptor.
 
 link:#a3208[See
-Java EE Application XML Schema Structure] shows a graphic representation
-of the structure of the Java EE application XML schema.
+Jakarta EE Application XML Schema Structure] shows a graphic representation
+of the structure of the Jakarta EE application XML schema.
 
 
 
-=== [[a3208]]Java EE Application XML Schema Structure
+=== [[a3208]]Jakarta EE Application XML Schema Structure
 
 image:Platform_Spec-11.png[image]
 
-=== Common Java EE XML Schema Definitions
+=== Common Jakarta EE XML Schema Definitions
 
 The XML Schema located at
-_http://xmlns.jcp.org/xml/ns/javaee/javaee_8.xsd_ defines types that are
-used by many other Java EE deployment descriptor schemas, both in this
+_http://xmlns.jcp.org/xml/ns/jakartaee/jakartaee_8.xsd_ defines types that are
```

Comment:
this URL will be wrong unless the jcp is going to host our xsds!!

Reply:
I realised this and reverted my changes and raised an issue https://github.com/eclipse-ee4j/jakartaee-platform/issues/45